src/error.rs

```diff
@@ -42,6 +42,8 @@ pub enum ClientErrorKind {
     NoAuthenticator,
     /// Required parameter was not provided
     MissingParam,
+    /// Error while using the SPIFFE Workload API
+    Spiffe,
```

Comment:
If the Spiffe error object implements `std::error::Error` you could/should add it here like we do with `Interface` above. If it doesn't.... you should probs put that on the TODO list 🤡

Reply:
Ah yes good point... There is an error, I will put it there.